eks-distro-base/Makefile

```diff
@@ -253,7 +253,7 @@ define UPDATE_TARGET_BODY
 	$(eval CHECK_UPDATE_RESULT:=$(shell cat $(call CHECK_UPDATE_FILE_PATH,$(IMAGE_COMPONENT),$(VERSIONED_VARIANT))))
 	$(if $(filter error,$(CHECK_UPDATE_RESULT)),@exit 1,)
 	$(if $(filter none,$(CHECK_UPDATE_RESULT)),@echo No updates for $(@),)
-	$(if $(filter updates,$(CHECK_UPDATE_RESULT)),make $(UPDATE_TARGET) && @echo true > $(MAKE_ROOT)/$(IMAGE_NAME)-pushed)
+	$(if $(filter updates,$(CHECK_UPDATE_RESULT)),make $(UPDATE_TARGET) && echo true > $(MAKE_ROOT)/$(IMAGE_NAME)-pushed)
```

Comment:
The @ was an issue since this was running a bash context since it was the first command of the shell.